cypress/integration/home.spec.ts

```diff
@@ -39,7 +38,7 @@ describe('Home screen', { defaultCommandTimeout: 30000 }, () => {
     });
   });
 
-  context('when clicking on the random button', () => {
+  xcontext('when clicking on the random button', () => {
```

Comment:
@hoangmirs Removing this because it cannot see the button some amount of times or you don't this to be tested anymore?

Reply:
I removed as I'm changing styles, adding in the next PR 🙇